src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/ThreadPoolMetricsCollector.java

```diff
@@ -186,8 +168,8 @@ public ThreadPoolStatus() {
             this.rejected = -1;
             this.threadsCount = -1;
             this.threadsActive = -1;
-            this.queueLatency = null;
-            this.queueCapacity = null;
+            this.queueLatency = -1.0;
+            this.queueCapacity = -1;
```

Comment:
We might need to set to the queueCapacity value right?

Reply:
this is the default setter for unit test. so we set all fields in this class to -1

Reply:
Got it